doc/dev/installation.rst

```diff
@@ -414,3 +414,94 @@ To generate html files for the documentation for Catalyst:
   make docs
 
 The generated files are located in ``doc/_build/html``
+
+Install a Frontend-Only Development Environment from TestPyPI Wheels
+--------------------------------------------------------------------
+
+It is possible to work on the source code repository and test the changes without 
+having to compile Catalyst. This is ideal for situations where the changes do not target the 
+runtime or the MLIR infrastructure, and only concern to the frontend. It basically 
+makes use of the shared libraries already shipped with the TestPyPI Catalyst wheels.
+
+Essential Steps
+^^^^^^^^^^^^^^^
+
+.. tabs::
+
+   .. group-tab:: Full CPL Suite
+
+      To activate the development environment, open a terminal and issue the following commands:
+
+      .. code-block:: console
+
+        # Clone the Catalyst repository  
+        git clone --recurse-submodules --shallow-submodules https://github.com/PennyLaneAI/catalyst.git
+
+        # Activate the development environment based on the latest TestPyPI wheels.
+        # First argument is the name of the Python environment
+        # Second argument is the type of Wheel installation. 
+        # 'full' installs the whole CPL suite
```

Comment:
The reader might not know what CPL means

Reply:
True. Done.